src/constants/bar/modes-options-bar.js

```diff
@@ -14,12 +14,12 @@ export const modes = {
 }
 
 export const tooltips = {
-  pureRandom: "Generate random data.",
-  configurableRandom: "Generate random data with ability to configure range, number of series and categories.",
-  balanceSimulation: "Analyze expenses and income.",
-  symbolsAnalysis: "Analyze the quantity of symbols in text.",
-  wordsAnalysis: "Analyze the quantity of words in text.",
-  interestingFacts: "Some very interesting facts from our life."
+  pureRandom: "In this mode, you have an opportunity to play with different configurations of the bar chart. Use checkboxes to customize the view and press the button to apply changes.",
```

Comment:
I'm not sure, but comma in text `In this mode, you have ...` coma is not obligatory.

Reply:
Thanks for your comment. Actually, for me also it seems like this comma is not needed, but I am using Grammarly to check my English and it says that I should put a comma there:
![image](https://user-images.githubusercontent.com/26466644/32160913-fcb4c250-bd53-11e7-9f11-a919648a8782.png)